bot/constants.py

```diff
@@ -563,6 +556,7 @@ class _URLs(_BaseURLs):
 
     site_staff: str = "".join([BaseURLs.site_schema, BaseURLs.site, "/staff"])
     site_paste = "".join(["paste.", BaseURLs.site])
+    site_api = "".join([BaseURLs.site, "/api"])
```

Comment:
Could you clarify this change? This would not have the correct value in production and require overriding. I prefer to remain consistent and just have prod values, and require devs override.

Reply:
I'm not sure I understand what you mean by [this](https://github.com/python-discord/bot/pull/2408#discussion_r1117955608)

Are you talking about the name of the class itself and having its own prefix ?
Or are you talking about passing the `env` kwarg to specify how to fetch it from env variables ?
Or none of the above ?

Reply:
Reverted in 686590c0d797aea28d79d94ea939277f734f404d

I got confused & misunderstood something while I was updating the docs. I think my brains got mangled with all of the renaming of the configs & I ticked in my head that we were constructing the site_api in that fashion.

My bad..